zerver/lib/actions.py

```diff
@@ -3799,11 +3799,26 @@ def do_update_message(user_profile: UserProfile, message: Message, topic_name: O
     }  # type: Dict[str, Any]
     changed_messages = [message]
 
+    send_event_users = list()  # call update event for these users
+    ums = UserMessage.objects.filter(message=message.id)
+    ums_user_ids = set(user.user_profile_id for user in ums)
```

Comment:
So, I think this change needs further work, because of security issues for a stream with `history_public_to_subscribers` False; message to those streams shouldn't get message-edit events for messages that they didn't receive.